### DISCUSSION TITLE
Test a live streaming site with thousands of users watching a video.

### DISCUSSION THREAD

**Discussion by alimardani** [artilleryio/artillery#1138] [Artillery]
Hi Team,

Is the following test possible to be done by artillery

- Open an site (URL, no authentication needed) which shows live streaming video.
- Simulate thousands of virtual users lets say 20k for example.
- Do the test for 10 minutes.

Thanks

**Comment by hassy** [comment]
Yes it should possible. https://github.com/artilleryio/artillery-plugin-hls is a good place to start.
